Oxford University Press.js

```diff
@@ -113,26 +154,27 @@ var testCases = [
 						"creatorType": "author"
 					}
 				],
-				"date": "24 July 2003",
-				"ISBN": "978-0-19-322103-1",
-				"abstractNote": "Tutor Easy Piano accompaniments to selected pieces in Flute Time 1",
+				"date": "2003-07-24",
+				"ISBN": "9780193221031",
+				"abstractNote": "Piano accompaniments to selected pieces in Flute Time 1",
 				"libraryCatalog": "Oxford University Press",
 				"numPages": "32",
-				"publisher": "Flute Time",
-				"attachments": [],
-				"tags": [
-					"Flute",
-					"Music",
-					"Woodwind & Brass"
+				"publisher": "Oxford University Press",
```

Comment:
Obviously no argument on the timeliness of citing publisher places, but as you say -- as long as it's widely required, we should provide the metadata. In my experience Oxford, New York is the most appropriate -- I'm seeing this most often and it corresponds to those two top listed cities (not arguing with any of your other comments, I'm just being pragmatic). If you're OK with that, would you put it in?